modules/gapi/misc/python/test/test_gapi_infer_ov.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+
+import numpy as np
+import cv2 as cv
+import os
+import sys
+import unittest
+
+from tests_common import NewOpenCVTests
+
+
+try:
+
+    if sys.version_info[:2] < (3, 0):
+        raise unittest.SkipTest('Python 2.x is not supported')
+
+
+    openvino_is_available = True
+    try:
+        from openvino.runtime import Core, Type, Layout, PartialShape
+        from openvino.preprocess import ResizeAlgorithm, PrePostProcessor
+    except ImportError:
+        openvino_is_available = False
+
+
+    def skip_if_openvino_not_available():
+        if not openvino_is_available:
+            raise unittest.SkipTest("OpenVINO isn't available from python.")
+
+
+    class test_gapi_infer_ov(NewOpenCVTests):
+
+        def test_age_gender_infer_image(self):
+            skip_if_openvino_not_available()
+
+            root_path  = '/omz_intel_models/intel/age-gender-recognition-retail-0013/FP32/age-gender-recognition-retail-0013'
+            model_path = self.find_file(root_path + '.xml',   [os.environ.get('OPENCV_DNN_TEST_DATA_PATH')])
```

Comment:
Could we utilize other models here? E.g. `.onnx`?

Reply:
We're using a couple of them for `ONNX` backend tests.